contracts/manifold/lazyclaim/ERC721LazyPayableClaim.sol

```diff
@@ -299,8 +368,19 @@ contract ERC721LazyPayableClaim is IERC165, IERC721LazyPayableClaim, ICreatorExt
      */
     function tokenURI(address creatorContractAddress, uint256 tokenId) external override view returns(string memory uri) {
         TokenClaim memory tokenClaim = _tokenClaims[creatorContractAddress][tokenId];
-        require(tokenClaim.claimIndex > 0, "Token does not exist");
-        Claim memory claim = _claims[creatorContractAddress][tokenClaim.claimIndex];
+        Claim memory claim;
+        uint256 mintOrder;
+        if (tokenClaim.claimIndex != 0) {
```

Comment:
same thing here, I expected that by incurring the extra upfront cost to store CC version on the claim we would use that as the condition, rather than just having fallback behavior